docs/content/faq.md

```diff
@@ -144,5 +144,39 @@ The Azure mixins are going to be full fledged tools that communicate with the
 Azure APIs, and provide an improved user experience. It all depends on how much
 time you want to invest, and what you are starting from.
 
+## Does the Porter has its own Registry like DockerHub, ACR,Quay etc?
+
+ No, Porter does have its Registry, rather use an OCI compliant artifact store. And now DockerHub is also OCI compliant registry. The list of registries that works with CNAB ie bundles [here](https://porter.sh/compatible-registries/) .
+
+## What is the use of Mixins?
```

Comment:
Let's make the question more tailored to the answer provided, and make the question sound like confrontational. 😀 

```suggestion
## What do mixins do?
```